src/react-apps/applications/ux-editor/src/utils/arrayHelpers/arrayLogic.ts

```diff
@@ -0,0 +1,110 @@
+import { createSelector } from 'reselect';
+
+export interface IUpdateArrayObj {
+  array: any[];
+  object: any;
+}
+
+export interface ITwoArraysObj {
+  array: any[];
+  order: any [];
+}
+
+const pushOrPop = (obj: IUpdateArrayObj) => {
```

Comment:
You can deconstruct in the parameter list
`const pushOrPop = ({array, object}: IUpdateArrayObj) => { .... }`

Reply:
So you access the array with the name `array` and not have to access it from `obj.array`.